SetupScript.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+if [ $EUID -ne 0 ]; then
+	echo "Please run as root user"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c1 google.com > /dev/null
+
+if [ $? -ne 0 ]; then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git /opt/Epsilon-Raspberry/
+cp /opt/Epsilon-Raspberry/primary/xorg.conf /etc/X11/ 
+cp /opt/Epsilon-Raspberry/primary/config.txt /boot/
+
+#install rabbit-mq
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | tee /etc/apt/sources.list.d/rabbitmq.list && apt-get update && apt-get install rabbitmq-server
+apt-get -y install cmake \
+	libboost-dev \
+	openssl \
+	libssl-dev \
+	libblkid-dev \
+	e2fslibs-dev \
+	libboost-all-dev \
+	libaudit-dev \
+	software-properties-common \
+	build-essential \
+	mesa-common-dev \
+	libgl1-mesa-dev
+
+if [ -n "/usr/local/lib/librabbitmq" ]; then
+	echo "Rabbitmq already setup"
+else
+	git clone https://github.com/alanxz/rabbitmq-c /tmp/rabbitmq-c/
+	mkdir /tmp/rabbitmq-c/build
+	(cd /tmp/rabbitmq-c/build && cmake .. && cmake --build .)
+	cp /tmp/rabbitmq-c/build/librabbitmq/*.a /usr/local/lib/
+	cp /tmp/rabbitmq-c/build/librabbitmq/*.so* /usr/local/lib/
+fi
+
+#begin QT install
+apt-get update
```

Comment:
Update was already ran in this script, we can remove this and it'll make save some time in running this script (albiet not much in the grand scheme of it cause it's compiling QT haha)